plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -291,6 +291,24 @@ static function ( string $error_message ): string {
 		</div>
 		<div class="clear"></div>
 	</div>
+	<p>
```

Comment:
Can we use `italic` font style same as WP footer text `Thank you for creating with WordPress`?

Reply:
I don't think so, because then we'd have italics stacked on italics. Also, the version on the right isn't in italics:

![image](https://github.com/user-attachments/assets/1e189d13-ec2b-42bd-843f-bc7a00e2b2ac)

I think the italics font is supposed to convey some kind of personal note.